bundle/libraries/libraries.go

```diff
@@ -173,7 +173,7 @@ func IsLocalPath(path string) bool {
 		return false
 	}
 
-	return !isWorkspacePath(path)
+	return !isWorkspacePath(path) && !isReposPath(path)
```

Comment:
Can't we assume that local paths are always relative, thus pushing down the absolute path check here?

Otherwise, we might need to perform the absolute check in other spots as well.

Reply:
Good point, I think we indeed can and should actually as we don't allow absolute path from local file system anyway

Reply:
Do we still need `isWorkspacePath` and/or `isReposPath` now?